index.bs

```diff
@@ -2097,15 +2099,15 @@ the value is the IDL {{undefined}} value.
              :  {{DOMString}}
              :  an [=enumeration=] type
              :: The value of the <emu-t class="regex"><a href="#prod-string">string</a></emu-t> token
-                is the sequence of 16 bit unsigned integer code units
-                (hereafter referred to just as <dfn id="dfn-code-unit">code units</dfn>)
+                is the sequence of 16 bit unsigned integer [=code units=]
                 corresponding to the UTF-16 encoding of |S|.
```

Comment:
I think we can just say that the value is |S|. We already know the value is a scalar value string from how we collect it. And those are compatible with strings.

Reply:
Done.